src/hotspot/share/runtime/arguments.cpp

```diff
@@ -522,6 +522,7 @@ static SpecialFlag const special_jvm_flags[] = {
 #endif // X86
 
   { "HeapFirstMaximumCompactionCount", JDK_Version::undefined(), JDK_Version::jdk(24), JDK_Version::jdk(25) },
+  { "UseVtableBasedCHA",               JDK_Version::undefined(), JDK_Version::jdk(24), JDK_Version::jdk(25) },
```

Comment:
We don't usually put diagnostic VM options in this table.

Reply:
Ok, I was confused by the following comment before the table: 
```
 * To remove internal options (e.g. diagnostic, experimental, develop options), use
 * a 2-step model adding major release numbers to the obsolete and expire columns.
```
Does it make sense to adjust it as well?

Reply:
Oh, right, that is what the comment says. I'd forgotten about the 2-step process.